.github/workflows/dkg-run.yaml

```diff
@@ -0,0 +1,33 @@
+name: Deploy Canary Clusters
+
+on: 
+    workflow_dispatch:
+jobs:
+  deploy:
+    name: DKG run with AWS and GCP Relays
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - id: 'auth'
+      uses: 'google-github-actions/auth@v0'
+      with:
+        credentials_json: '${{ secrets.GOOGLE_CREDENTIALS }}'
+
+    - id: 'get-credentials'
+      uses: 'google-github-actions/get-gke-credentials@v0'
+      with:
+        cluster_name: ${{ secrets.GKE_CLUSTER }}
+        location: ${{ secrets.GKE_REGION }}
+        credentials: ${{ secrets.GOOGLE_CREDENTIALS }}
+        project_id: ${{ secrets.PROJECT_ID }}
+
+    - name: Run DKG with Relay 1
```

Comment:
Will we add 0 for the fun here too?

Reply:
yes, we will do, but just because it is the only prod relay atm, i thought lets do the drill on 1 and 2 for a while and then add 0